.cbmc-batch/jobs/aws_hash_callback_c_str_eq/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--unwindset;strcmp.0:129;--object-bits;8"
```

Comment:
`--object-bits;8` is default

Reply:
Makefile.common autogenerates this file: this should really be a change to Makefile.common, if we care.